src/agent/firebase-controller.ts

```diff
@@ -36,6 +36,9 @@ export class FirebaseController implements Controller {
   debuggeeId?: string;
   bpRef?: firebase.database.Reference;
 
+  markActiveInterval: ReturnType<typeof setInterval> | undefined;
+  markActivePeriod: number = 60 * 60 * 1000; // 1 hour in ms.
```

Comment:
Consider changing the variable name to `markActivePeriodMsec`, then there's no need to find the definition to find the comment indicating the units.